configuration/messageproperties/impl_en.properties

```diff
@@ -14,9 +14,6 @@ impl.nationality=Nationality
 impl.nationality.desc=Nationality
 impl.email=Email
 impl.email.desc=Primary email address
-Khmer_Given_Name=Name in Khmer
```

Comment:
Those are backend messages and I think are needed. I don't recall exactly why but I am pretty sure that we have done that in purpose. Unless you specifically think that this generates a bug somehow, I suggest we keep those messages.